helpers/DeployHelper.ts

```diff
@@ -1085,6 +1086,59 @@ export class DeployHelper {
     };
   }
 
+  async deployMCPayment(
+    ownerPercentage: number,
+    deployStrategy: "basic" | "create2" = "basic",
+  ): Promise<{
+    mcPayment: Contract;
+  }> {
+    if (ownerPercentage < 0 || ownerPercentage > 100) {
+      throw new Error("Owner percentage should be between 0 and 100");
+    }
+    const owner = this.signers[0];
+    const MCPaymentFactory = await ethers.getContractFactory("MCPayment");
+    const Create2AddressAnchorFactory = await ethers.getContractFactory("Create2AddressAnchor");
+
+    let mcPayment;
+    if (deployStrategy === "create2") {
+      this.log("deploying with CREATE2 strategy...");
+
+      // Deploying MCPayment contract to predictable address but with dummy implementation
+      mcPayment = (
+        await ignition.deploy(MCPaymentModule, {
```

Comment:
`MCPaymentProxyModule` but not `MCPaymentModule` should be here. You don't need `MCPaymentModule` at all.